### ISSUE TITLE
Something Funky is going on. Pcsv is installed, but no commands are available.

### ISSUE THREAD

**Issue by anoduck** [mhayashi1120/Emacs-pcsv#2]
So, pcsv is definitely installed, but there are no commands available to use it. Pcsv-mode isn't even available. Could this indicate pcsv is a library and does not provide a mode for use?

**Comment by kisaragi-hiu** [comment]
Yes, this is a library for parsing CSV files into Emacs values, just like the builtin json.el provides a library for parsing JSON files. This library does not provide a mode, nor is that its stated purpose.

**Comment by anoduck** [comment]
> __This library__...

Ah... Gotcha... Cool...
